packages/formation-react/src/components/RightRail/RightRailBlock.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SidebarBlock = props => {
+  let headingContent = (
+    <h2 className="vads-u-margin-top--0 vads-u-padding-bottom--1p5 vads-u-border-bottom--3px vads-u-border-color--primary sidebar-heading">
+      {props.heading}
+    </h2>
+  );
+  switch (props.headingLevel) {
+    case 3:
+      headingContent = (
+        <h3 className="vads-u-margin-top--0 vads-u-padding-bottom--1p5 vads-u-border-bottom--3px vads-u-border-color--primary sidebar-heading">
+          {props.heading}
+        </h3>
+      );
+      break;
+    case 4:
+      headingContent = (
+        <h4 className="vads-u-margin-top--0 vads-u-padding-bottom--1p5 vads-u-border-bottom--3px vads-u-border-color--primary sidebar-heading">
+          {props.heading}
+        </h4>
+      );
+      break;
+    default:
+    // dont change anything
+  }
```

Comment:
`headingContent ` could be initialized to null, then the `default` case here could be a good case for the `h2`. Potentially we could also store the `className` value in its own var so that long string doesn't have to be repeated.

Reply:
Does the class `.sidebar-heading` need to be styled at all?

Reply:
Yep, great idea. Just updated this. The Sidebar heading is already styled from the default heading styles on VA.gov so that it is inherited. The `.sidebar-heading` class is in case anyone needs to tap into it to style it differently.

Reply:
We _may_ want to update it to follow the sass naming conventions of this repo - `vads-c-right-rail` might be a good name. https://design.va.gov/documentation/naming